src/components/board/board.ts

```diff
@@ -11,6 +11,12 @@ export interface Tick {
   time: number;
 }
 export type Duration = PBDuration.AsObject;
+export interface AnalogValue {
```

Comment:
can we just make this `export type AnalogValue = pb.ReadAnalogReaderResponse.AsObject`? I'm not 100% sure it'll work but I would like to typealias it since it doesn't introduce any new values

Reply:
it works, changed